distributed/tests/test_worker_state_machine.py

```diff
@@ -985,17 +985,17 @@ async def get_data(self, comm, *args, **kwargs):
 
 @gen_cluster()
 async def test_deprecated_worker_attributes(s, a, b):
-    n = a.state.comm_threshold_bytes
+    n = a.state.target_message_size
```

Comment:
This tests the use of a deprecated attribute with the same name on the `WorkerState`. Since we renamed `comm_threshold_bytes`, I had to pick another variable.